.gitignore

```diff
@@ -5,4 +5,5 @@
 /dist/
 __pycache__/
 liquidctl/_version.py
-.DS_Store
\ No newline at end of file
+.DS_Store
+.vscode
```

Comment:
Please use a personal global gitignore file for editor files, given that they are a personal and therefore vary varied choice.  See [`gitignore`](https://git-scm.com/docs/gitignore) for more information, especially the `core.excludesFile` part.

Reply:
It's done.